migrate/20250110_add_kubernetes_models.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:kubernetes_cluster) do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
```

Comment:
Since we use UBIDs, we don't set a default UUID at the database level for `id` columns

https://github.com/ubicloud/ubicloud/blob/main/migrate/20230721_remove_default_ids.rb